custom-words.txt

```diff
@@ -3956,3 +3956,21 @@ protobuf
 proto
 msvideo
 SOQL
+businesscases
```

Comment:
you usually shouldn't need to add 'words' like this to custom-words because they're not words - and you could be camelcasing them, such as 'businessCases' instead

Reply:
What is wacc?

Please try replacing your list with just the below:

(adding one of webappCollectors or webAppCollectors, but not both)

(Also note that its been spelled 'datacenter', not 'datacentre' in all our other REST API specs. I think this is the usual american spelling? Anyway, let's stay consistent across Azure.)

and I don't think we want to introduce EUAP terminology if we don't have to -- it seems like we've avoided doing so far!

```
avsSummaries
iaasSummaries
paasSummaries
webAppCollectors
wacc
hyperthread
capex
opex
decomission
decommisioned
```